pkg/frontend/querymiddleware/codec_test.go

```diff
@@ -346,17 +346,83 @@ func TestMetricsQuery_MinMaxTime_TransformConsistency(t *testing.T) {
 			initialMetricsQuery: instantRequest,
 			updatedStartTime:    &updatedEndTime,
 			updatedEndTime:      &updatedEndTime,
-			updatedQuery:        "",
 
 			expectedUpdatedMinT: updatedEndTime.UnixMilli(),
 			expectedUpdatedMaxT: updatedEndTime.UnixMilli(),
 			expectedErr:         nil,
```

Comment:
Given `expectedErr` is `nil` for both cases and not used in the test body below, can we remove it altogether?